src/Api/AdminConsole/Controllers/OrganizationUsersController.cs

```diff
@@ -230,29 +225,22 @@ private ListResponseModel<OrganizationUserUserDetailsResponseModel> GetResultLis
 
 
     [HttpGet("{id}/groups")]
-    public async Task<IEnumerable<string>> GetGroups(string orgId, string id)
+    [Authorize<ManageGroupsOrUsersRequirement>]
+    public async Task<IEnumerable<string>> GetGroups(string id)
     {
         var organizationUser = await _organizationUserRepository.GetByIdAsync(new Guid(id));
```

Comment:
Can you please update all instances of `string id` or `string orgId` parameters to be `Guid`s while you're here?